modules/op-release-notes-1-0.adoc

```diff
@@ -72,6 +72,45 @@ For this release, use flags `disable-home-env-overwrite` and `disable-working-di
 == Known issues
 * If you are upgrading from an older version of {pipelines-title}, you must delete your existing deployments before upgrading to {pipelines-title} version 1.0. To delete an existing deployment, you must first delete Custom Resources and then uninstall the {pipelines-title} Operator. For more details, see the uninstalling {pipelines-title} section.
 * Submitting the same `v1alpha1` Tasks more than once results in an error. Use `oc replace` instead of `oc apply` when re-submitting a `v1alpha1` Task.
+* The `buildah` ClusterTask does not work when a new user is added to a container.
++
+During Operator installation, the `--storage-driver` flag for the `buildah` ClusterTask is not specified, therefore the flag is set to its default value. In some cases, this may cause the storage-driver to be set incorrectly. When a new user is added, the incorrect storage-driver results in the failure of `buildah` ClusterTask with the following error:
++
+[source, bash]
+----
+useradd: /etc/passwd.8: lock file already used
+useradd: cannot lock /etc/passwd; try again later.
+----
++
+As a workaround, you can manually set the `--storage-driver` flag to `overlay` value in the `buildah-task.yaml` file:
++
+. Login to your cluster as a `cluster-admin`:
++
+----
+$ oc login -u <login> -p <password> https://openshift.example.com:6443
+----
+. Use the `oc edit` command to edit `buildah` ClusterTask:
++
+----
+$ oc edit clustertask buildah
+----
++
+The current version of the `buildah` clustertask YAML file opens in the editor set by your `EDITOR` environment variable.
```

Comment:
in the editor set by your `EDITOR` environment variable.>in an editor? For this task, do we need to mention why a particular editor opens up?

Reply:
I was told to explicitly mention that file will open up in the editor that user has set in the Editor environment variable. Some users may not be aware of it.